src/main/java/org/reaktivity/nukleus/oauth/internal/OAuthRealms.java

```diff
@@ -28,19 +28,25 @@
 
 import org.jose4j.jwk.JsonWebKey;
 import org.jose4j.jwk.JsonWebKeySet;
+import org.jose4j.jws.JsonWebSignature;
+import org.jose4j.jwt.JwtClaims;
+import org.jose4j.jwt.consumer.InvalidJwtException;
 import org.jose4j.lang.JoseException;
 import org.reaktivity.nukleus.internal.CopyOnWriteHashMap;
 
 public class OAuthRealms
 {
+    public static final String[] EMPTY_STRING_ARRAY = new String[0];
+
+    private static final String SCOPES_CLAIM = "scopes";
```

Comment:
JWT claim name is `scope`, not `scopes`.
See https://tools.ietf.org/id/draft-bertocci-oauth-access-token-jwt-00.html#rfc.section.3

Change to `private static final String SCOPE_CLAIM = "scope";`

Note: this may require corresponding changes to some spec scripts if they are currently using `scopes` instead of `scope` as the claim name.